lib/shared/messages.js

```diff
@@ -35,7 +35,9 @@ module.exports = {
     flashComplete: _.template([
       '<%= imageBasename %> was successfully written to',
       '<%= drive.description %> (<%= drive.displayName %>)'
-    ].join(' '))
+    ].join(' ')),
+
+    flashProgress: _.template('<%= type === "check" ? "Validating" : "Flashing" %> <%= percentage %>%')
```

Comment:
Also, this has the %age after the word, whereas `progress-status.js` currently has the %age before the word?